ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -1693,6 +1702,22 @@ private class JdbcLedgerDao(
       ()
     }
 
+  private val SQL_DELETE_COMMAND = SQL("""
+     |delete from participant_command_submissions
+     |where deduplicate_key < {deduplicationKey}
+    """.stripMargin)
+
+  override def stopDeduplicatingCommand(
+      commandId: domain.CommandId,
+      submitter: Party): Future[Unit] =
+    dbDispatcher.executeSql("remove_expired_deduplication_data") { implicit conn =>
```

Comment:
The description here looks like a better name for the method.

Reply:
Oh, copy and paste error, there's a different method for removing expired deduplication data.

Reply:
For reference, `stopDeduplicatingCommand` removes a deduplication entry prior to its expiry. Entries should be removed when the commands are rejected.